Marlin/Marlin_main.cpp

```diff
@@ -1713,6 +1718,26 @@ static void homeaxis(AxisEnum axis) {
       }
     #endif
 
+    #ifdef BABYSTEPPING
+    if(axis == Z_AXIS)
+    {
+      baby_max_endstop[axis] = Z_BABY_DEFAULT_MAX_POS;
+      baby_min_endstop[axis] = Z_BABY_DEFAULT_MIN_POS;
+    }
+    #ifdef BABYSTEP_XY
+    else if(axis == X_AXIS)
+    {
+      baby_max_endstop[axis] = X_BABY_DEFAULT_MAX_POS;
+      baby_min_endstop[axis] = X_BABY_DEFAULT_MIN_POS;
+    }
+    else if(axis == Y_AXIS)
+    {
+      baby_max_endstop[axis] = Y_BABY_DEFAULT_MAX_POS;
+      baby_min_endstop[axis] = Y_BABY_DEFAULT_MIN_POS;
+    }
+    #endif //BABYSTEP_XY
+    #endif //BABYSTEPPING
+
```

Comment:
I added the indentation, I personally prefer the open curly brackets to have their own line, I think it makes the blocks/nests clearer.